test/unit/api/properties.js

```diff
@@ -184,6 +184,36 @@ describe('api/prop', () => {
     });
   });
 
+  describe('object', () => {
+    let elem;
+
+    beforeEach((done) => {
+      elem = create(prop.object());
+      afterMutations(done, 1);
```

Comment:
I know you got this call from somewhere else, but it doesn't take the `1` argument. Just need `done`.